pysrc/pip_resolve.py

```diff
@@ -309,9 +456,15 @@ def create_dependencies_tree_by_req_file_path(requirements_file_path,
 
         # build a tree of dependencies
         package_tree = create_tree_of_packages_dependencies(
-            dist_tree, top_level_requirements, requirements_file_path, allow_missing, only_provenance)
-        print(json.dumps(package_tree))
+            dist_tree,
+            top_level_requirements,
+            optional_dependencies,
+            requirements_file_path,
+            allow_missing,
+            only_provenance
+        )
 
+        print(json.dumps(package_tree))
```

Comment:
☝🏻

Reply:
Yeap, that's the way the actual result gets back to TypeScript, would you like me to add a comment to reflect that ? Or ?

Reply:
Oh. Yes, I thought it was debugging. Sorry 🤦🏻